src/content/getting-started/designers/designers.md

```diff
@@ -25,58 +25,57 @@ title: Designers
+The Carbon Design Kit is a constantly evolving Sketch document containing all of the design system’s visual assets, including components, icons, color palettes, grids, etc. The kit helps you work faster, with confidence that your product or experience is visually aligned with the IBM Design System. It utilizes symbols and text styles, organized via pages in the left side panel of Sketch. When used in conjunction with this site, the Carbon Design Kit provides designers and developers with everything they need to get up and running with Carbon.
 
-### Release Schedule
+### Release schedule
 
-Every six months the Carbon team releases a major update to the system, including a new design kit, along with new version of the components. Style refinement and bug fixes are released as continuous updates that happen throughout the year. We track all release updates in our [change log](https://github.com/carbon-design-system/carbon-design-kit/releases).
+The Carbon core team releases a major update to the system approximately twice a year, including a revised design kit and new versions of the components when needed. Style refinements and bug fixes are released as continuous updates throughout the year. All design kit release updates are tracked in the [change log](https://github.com/carbon-design-system/carbon-design-kit/releases).
 
 ## How to get it
 
-There are three ways to obtain the kit. For timely updates, install the kit as Sketch Shared Library. Please note this requires Sketch version 53 or above.
+There are three ways to obtain the kit. For timely updates, install the kit as Sketch Shared Library. _Please note: this method requires Sketch version 53 or above._
 
 <grid-wrapper col_lg="8" flex="true" bleed="true">
 <clickable-tile
-    title="Add Carbon design kit to Sketch library (Beta)"
+    title="Add Carbon Design Kit to Sketch library (Beta)"
     href="sketch://add-library/cloud/JaVzz"
     type="resource"
      >
-    <img src="images/sketch-icon.png" alt="Carbon design kit Sketch Shared Library"  />
+    <img src="images/sketch-icon.png" alt="Carbon Design Kit Sketch Shared Library"  />
 </clickable-tile>
 </grid-wrapper>
 
-If you might work with a proprietary add-on library and still want to be connected for updates, please set up the kit via the Box Drive by following [these instructions](#get-the-kit-via-box).
+If you work with a proprietary add-on library and still want to be connected for updates, set up the kit via Box Drive by following [these instructions](#get-the-kit-via-box). (Internal IBM users only)
```

Comment:
This section is a little off. The tile actually takes a user to the Sketch Cloud download/sync thing that ANYONE can use, whether and IBMer or not. Its a beta function right now just because we haven't done a lot testing with it. Jeoff will like this to be the preferred method of connecting to the library once we test it a little more.

Connecting through Box Drive is only for internal IBMers.

I don't think we should say "If you would rather not receive library updates," I think rephrasing to something like "For one time downloads of the kit, ...". The catch with GitHub is you have to keep going back to it to get the updates, the updates are there they're just not automatic.

Reply:
I'm not totally clear on what needs to be changed here. The tile goes with the first paragraph, maybe that is the confusion? And the Box Drive option already stipulates IBM internal only. . . 

The other note makes sense, I can change that. :)

Reply:
It wasn't very clear to me that that was the first way and the tile was associated with the content above it and not below it. The other two get a section of its own where as this one is wrapped up in the summary section.

Reply:
Yeah, I think in this case the tile might add more confusion than value to the flow of the page. Going to just change that to a link for clarity. Will also look at how those paragraphs are broken down.